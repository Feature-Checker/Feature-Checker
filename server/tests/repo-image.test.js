const request = require("supertest");

let res;
let req;

beforeEach(() => {
    req = {
        query: {}
    };
    res = {
        json: jest.fn(),
        send: jest.fn(),
        status: jest.fn()
    }
})

const {
    fetchImage
} = require('../src/controllers/repo-image');

describe('working fetchImage async function', () => {
    it('should do what I like', () => {
        expect(fetchImage(req, res)).toMatchSnapshot();
    })
})