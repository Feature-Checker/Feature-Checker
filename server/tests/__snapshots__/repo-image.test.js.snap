// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`working fetchImage async function should do what I like 1`] = `Promise {}`;
